cardano-cli/test/cardano-cli-golden/files/golden/conway/tx-three-votes-view.out.json

```diff
@@ -0,0 +1,65 @@
+{
+    "auxiliary scripts": null,
+    "certificates": null,
+    "collateral inputs": [],
+    "era": "Conway",
+    "fee": "185433 Lovelace",
+    "governance actions": [],
+    "inputs": [
+        "96e3be606ca03f67b0d299ca0eb0d660670edaa8452fffcc4bfccd8757a9b45d#0"
+    ],
+    "metadata": null,
+    "mint": null,
+    "outputs": [
+        {
+            "address": "addr_test1qp39w0fa0ccdc4gmg87puydf2kxt5mgt0vteq4a22ktrcssg7ysmx64l90xa0k4z25wpuejngya833qeu9cdxvveynfscsskf5",
+            "address era": "Shelley",
+            "amount": {
+                "lovelace": 49497089775
+            },
+            "network": "Testnet",
+            "payment credential key hash": "62573d3d7e30dc551b41fc1e11a9558cba6d0b7b179057aa55963c42",
+            "reference script": null,
+            "stake reference": {
+                "stake credential key hash": "08f121b36abf2bcdd7daa2551c1e6653413a78c419e170d3319924d3"
+            }
+        }
+    ],
+    "redeemers": {},
+    "reference inputs": [],
+    "required signers (payment key hashes needed for scripts)": null,
+    "return collateral": null,
+    "total collateral": null,
+    "update proposal": null,
+    "validity range": {
+        "lower bound": null,
+        "upper bound": null
+    },
+    "voters": {
```

Comment:
This field can contain multiple voters: `voters` is a map from voter to votes of this voter. This output shows a case where there is one voter, with three votes.